recipes/nsearch/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+mkdir build
+cd build
+
+cmake -Wno-dev ..
+cmake --build . --config Release
+
+make check
```

Comment:
Can we run these tests on Windows too?

Reply:
We can not. We are using MSVC compilers and in this case, cmake does not generate a Makefile